suricata.yaml.in

```diff
@@ -753,6 +753,7 @@ dpdk:
       # - auto takes all cores
       # in IPS mode it is required to specify the number of cores and the numbers on both interfaces must match
       threads: auto
+      power-saving: no # yes to switch to interrupt mode 
```

Comment:
Does DPDK refer to this as power saving? Or polling?  I'd use polling.  What is the common term for the non-polling mode? If power-hungry, then power-saving makes sense I guess. But in all seriousness, if we have "polling" mode, what is the default mode referred to as?

Reply:
Tbh, I feel like DPDK doesn't consistently name this feature. But indeed, in those few cases, interrupt mode can be more preferred option. 

There is one example app available that implements interrupts - https://doc.dpdk.org/guides/sample_app_ug/l3_forward_power_man.html but that also implements power management of cores (switching between C and P states). 
Then there is only epoll API doxygen page https://doc.dpdk.org/api/rte__epoll_8h.html

So my take was to make it more user-oriented to show the direct impact of what it can do when enabling it. But I can rename it to `interrupt-mode:` to make it closer to the underlying concept.